### PR TITLE
Edit a user profile basic information

### PR DESCRIPTION
**Pull request by gregory-latinier**
### Verify that
- [X] the branch name follows the git flow naming convention (feature/i18n)
- [X] the UT reference is appended to the branch name if it concerns a Jira Task (feature/i18n/UT-283)
- [X] the design is responsive
- [X] the code doesn't break Server Side Rendering
- [X] new functions are prefixed with JSDoc information
- [X] the pull request contains unit tests for new code that passes and has at least 50% coverage
- [X] the code doesn't break the api endpoints contract
- [X] JS Standard code style has been verified with ESLint

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_Update your profile_

* Change your information
* Save
* Reload

**Expected result:** You should see the updated values

_BONUS: Update the avatar_

* Change the avatar
* Save

**Expected result:** the avatar in the toolbar is also updated

### CODE REVIEW

**Comment by gregory-latinier** [comment]
I agree that we should provide a better way to deal with profile image and offer some kind of processing

**Comment by gregory-latinier** [comment]
The placeholder preview with the size in it are a hint for the user of what image size the system is expecting since we don't offer image processing for now

**Comment by icaroharry** [comment]
UI/UX suggestions:

1) The 'disabled' field is too similar to the fields with placeholders:
image

2) If we're using the cards, I suggest using a white background for them:
image

3) I suggest moving the "group" title to outside the cards. Similar to the zeplin mockup:
image

I like the idea of grouping the fields into a matrix. It's better than displaying a single column form :)

**Comment by gregory-latinier** [comment]
The disabled field will be dealt with when we go over the override quasar css that is not good and break some form components

I'll give it a try for the background and title

File: packages/client/src/pages/users/profile/profile.vue [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    },
+    images: {
+      avatarUrl: { required, url },
+      cover: { url }
+    },
+    skills: {}
+  },
+  async mounted () {
+    if (!this.user) {
+      this.$router.push({ path: `/login?returnUrl=${this.$route.path}` })
```
Comment (icaroharry) [inline packages/client/src/pages/users/profile/profile.vue]: There's no `/login` route in the client app. You should redirect to `${process.env.AUTH_DOMAIN}/${this.$route.params.locale}/login?returnUrl=${this.$route.path}`

File: packages/client/src/pages/users/profile/profile.vue [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    }
+  },
+  methods: {
+    ...mapActions('users', [
+      'fetchUserProfile',
+      'updateProfileMainInformation',
+      'updateProfileJob',
+      'updateProfileImages'
+    ]),
+    ...mapActions('auth', ['updateAvatarUrl']),
```
Comment (icaroharry) [inline packages/client/src/pages/users/profile/profile.vue]: why this action is not under the `users` module as well?

Comment (gregory-latinier) [reply packages/client/src/pages/users/profile/profile.vue]: Because the avatarUrl attribute is in the auth store.

**Pull request merged**